package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "metadata-service",
+  "version": "0.0.1",
+  "description": "Movie metadata microservice for usage by Kwiz",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kwizapp/metadata-service.git"
+  },
+  "maintainers": [
+    "Alex Scheitlin <alex.scheitlin@uzh.ch>",
+    "Roland Schlaefli <rolandschlaefli@gmail.com>",
+    "Nik Zaugg <nik.zaugg@uzh.ch>"
+  ],
+  "license": "MIT",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "micro src/index.js",
+    "test": "ava"
+  },
+  "dependencies": {
+    "dotenv": "^8.2.0",
```

Comment:
@kwizapp/dev-team I always find it best to **pin** all dependencies down to the patch version (without the ^ for ranges). We won't get any minor upgrades that might be unexpected, know exactly what is installed without looking into the look file, and dependabot can be used to update everything as needed anyways. 

What do you think? This would probably also need changes in some other repos.

Reply:
Makes sense! I'll change that. Thanks.